config/jobs/periodic/etcd/test-etcd-periodics.yaml

```diff
@@ -0,0 +1,46 @@
+periodics:
+  - name: periodic-etcd-test-ppc64le
+    cluster: k8s-ppc64le-cluster
+    decorate: true
+    decoration_config:
+      decoration_config:
+        bucket: ppc64le-kubernetes
+        path_strategy: explicit
+      gcs_credentials_secret: gcs-credentials
+    cron: "0 5/3 * * *"
+    branches:
+      - master
```

Comment:
is this needed?

Reply:
Right, we don't need this for periodic job.